packages/shared-ux/prompt/no_data_views/impl/src/no_data_views.component.tsx

```diff
@@ -7,95 +7,231 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import React from 'react';
 import { css } from '@emotion/react';
+import React from 'react';
 
-import { EuiEmptyPrompt, EuiPanel } from '@elastic/eui';
+import {
+  EuiButton,
+  EuiCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTextAlign,
+} from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { withSuspense } from '@kbn/shared-ux-utility';
 import { NoDataViewsPromptComponentProps } from '@kbn/shared-ux-prompt-no-data-views-types';
 
+import { i18n } from '@kbn/i18n';
 import { DocumentationLink } from './documentation_link';
-import { NoDataButtonLink } from './actions';
 
-// Using raw value because it is content dependent
-const MAX_WIDTH = 830;
+// max width value to use in pixels
+const MAX_WIDTH = 770;
 
-/**
- * A presentational component that is shown in cases when there are no data views created yet.
- */
-export const NoDataViewsPrompt = ({
+const PromptAddDataViews = ({
   onClickCreate,
   canCreateNewDataView,
   dataViewsDocLink,
-  onTryESQL,
-  esqlDocLink,
-  emptyPromptColor = 'plain',
-}: NoDataViewsPromptComponentProps) => {
-  const title = canCreateNewDataView ? (
-    <h2>
-      <FormattedMessage
-        id="sharedUXPackages.noDataViewsPrompt.youHaveData"
-        defaultMessage="You have data in Elasticsearch."
-      />
-      <br />
-      <FormattedMessage
-        id="sharedUXPackages.noDataViewsPrompt.nowCreate"
-        defaultMessage="Now, create a data view."
-      />
-    </h2>
-  ) : (
-    <h2>
-      <FormattedMessage
-        id="sharedUXPackages.noDataViewsPrompt.noPermission.title"
-        defaultMessage="You need permission to create data views"
-      />
-    </h2>
+  emptyPromptColor,
+}: Pick<
+  NoDataViewsPromptComponentProps,
+  'onClickCreate' | 'canCreateNewDataView' | 'dataViewsDocLink' | 'emptyPromptColor'
+>) => {
+  // Load this illustration lazily
+  const Illustration = withSuspense(
```

Comment:
I understand that this was here like this before the PR, but I just noticed that this pattern is faulty and only hurts. We should simplify to using the component directly: 

const icon = `<DataViewIllustration />`;


There is no need to code split because image inside is anyway loaded by url only when img tag is rendered. Also the code split is done incorrectly because React.lazy should be outisde the component


Could you please update in all place in these components?

Reply:
b15ba4f1698